### ISSUE TITLE
yolo v5 from pytorch hub not running on colab

### ISSUE THREAD

**Issue by HuakunShen** [ultralytics/yolov5#4841]
Pytorch Hub: https://pytorch.org/hub/ultralytics_yolov5/

Colab: https://colab.research.google.com/github/pytorch/pytorch.github.io/blob/master/assets/hub/ultralytics_yolov5.ipynb

The code in colab notebook won't run when running model inference.

Error:
```
Model Summary: 224 layers, 7266973 parameters, 0 gradients
Adding AutoShape... 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-8-0a0ec081fe3a> in <module>()
      8 
      9 # Inference
---> 10 results = model(imgs)
     11 
     12 # Results

7 frames
/usr/local/lib/python3.7/dist-packages/PIL/TiffImagePlugin.py in ImageFileDirectory_v2()
    685                 (TiffTags.FLOAT, "f", "float"),
    686                 (TiffTags.DOUBLE, "d", "double"),
--> 687                 (TiffTags.IFD, "L", "long"),
    688             ],
    689         )

AttributeError: module 'PIL.TiffTags' has no attribute 'IFD'
```

**Comment by glenn-jocher** [comment]
@HuakunShen thanks for the bug report! We will take a look at this, but the quick fix to restart your runtime as the message recommends.

**Comment by HuakunShen** [comment]
Sorry, you are right, restarting the runtime got it working.

**Comment by glenn-jocher** [comment]
@HuakunShen good news 😃! Your original issue may now be fixed ✅ in PR #4842. This relaxes Pillow dependency requirements that were causing the issue. Colab notebook should now work directly the first time with no issues!

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
